options/linux-headers/include/linux/vt.h

```diff
@@ -15,7 +15,7 @@
 
 #ifdef __cplusplus
 extern "C" {
-#endif __cplusplus
+#endif // __cplusplus
```

Comment:
This is not correct.

Reply:
This throws warnings/errors if left untouched, so the comment is necessary. [Here's](https://gcc.gnu.org/onlinedocs/cpp/Ifdef.html) what gcc says about it.

Reply:
Oh sorry, I completely misread, yeah this is correct.